apps/sertra/components/ServerDetail.module.css

```diff
@@ -0,0 +1,354 @@
+.container {
+  width: 100%;
+  display: flex;
+  column-gap: 2em;
+  flex-direction: column;
+}
+
+.headerRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.contentRow {
+  display: flex;
+  column-gap: 2em;
+}
+
+.columnLeft {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+.columnRight {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.descriptionContent,
+.row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0.75em 1em 12px 0.75em;
+  margin-bottom: 0.5em;
+  gap: 1em;
+  font-size: 16px;
+  line-height: 140%;
+}
+
+.descriptionContent,
+.row {
+  background: #242e48;
```

Comment:
My very recent PR introcuded CSS variables for our [theme colors](https://github.com/thunderstore-io/thunderstore-ui/pull/488/commits/d198b1c424d9a4f177193812c36bd237e650cc3d) and [basic fonts](https://github.com/thunderstore-io/thunderstore-ui/pull/488/commits/d078dab16f7b5c67e94861caca258ce944991a15), as well as changed from `px` to `rem` units. This PR looks like the CSS could be improved by adopting those things.